lib/thrift/protocols/binary.ex

```diff
@@ -53,10 +49,9 @@ defmodule Thrift.Protocols.Binary do
   def serialize(_, nil) do
     []
   end
-  def serialize(:bool, value) do
-    value = bool_to_int(value)
-    <<value::8-signed>>
-  end
+  def serialize(:bool, nil),   do: <<0::8-signed>>
```

Comment:
Yeah good point. If we're not sure, it might be better to start strict, true/false only. We can become less strict later without breaking applications.